.github/compatibility-test-matrices/release-v8.0.x/transfer-chain-a.json

```diff
@@ -0,0 +1,30 @@
+{
+  "chain-a": [
+    "release-v8.0.x"
+  ],
+  "chain-b": [
+    "v7.3.0",
+    "v7.2.1",
+    "v7.2.0",
+    "v6.2.0",
+    "v6.1.1",
+    "v5.3.1",
+    "v5.2.1",
+    "v4.4.2",
+    "release-v8.0.x"
+  ],
+  "entrypoint": [
+    "TestTransferTestSuite"
+  ],
+  "test": [
+    "TestMsgTransfer_Succeeds_Nonincentivized",
+    "TestMsgTransfer_Fails_InvalidAddress",
+    "TestMsgTransfer_Timeout_Nonincentivized",
+    "TestMsgTransfer_WithMemo",
+    "TestSendEnabledParam",
+    "TestReceiveEnabledParam"
```

Comment:
just for what it is worth, there are 16 unnecessary tests run here, but I guess it is okay